img.py

```diff
@@ -30,8 +37,14 @@ def __init__(self):
 		self.pt0 = 0,0
 		self.pt1 = 0,0
 		self.pts = []
-		self.imgW = 1200
-		self.imgH = 800
+		self.imgW = imgW
+		self.imgH = imgH

```

Comment:
Where are we getting this imgW and imgH from? Maybe specify something in comments for future developers.